web/src/sidebar_ui.js

```diff
@@ -144,7 +144,7 @@ export function initialize_left_sidebar() {
             user_settings.web_home_view === settings_config.web_home_view_values.all_messages.code,
         is_recent_view_home_view:
             user_settings.web_home_view === settings_config.web_home_view_values.recent_topics.code,
-        hide_unread_counts: settings_data.should_mask_unread_count(),
+        hide_unread_counts: settings_data.should_mask_unread_count(false),
```

Comment:
@karlstolley FYI can you confirm that's the intended logic from a376954a7a2348d44abed6d7752bbe5d3b04f268?

Reply:
I went for `false` because `undefined` is falsey, but yeah a confirmation would be great

Reply:
Sorry to have completely missed this @-mention. Yes, that appears to me to be the intended logic.